### ISSUE TITLE
Block, Block Section - Ensure ARIA roles, states, and properties are valid - (2036569798)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7782]
### Violation: 
 Ensure ARIA roles, states, and properties are valid 

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Severity: 
 5 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: Block, Block Section - Ensure ARIA roles, states, and properties are valid - (2036569798) 
- Module: Block, Block Section 
**Issue**
There are SECTION elements with invalid ARIA attribute "aria-expanded".

**User Impact**
Assistive technologies may be unable to parse this content.

**Code Reference** 
 ```
<section aria-expanded="true" aria-labelledby="867dceb1-e661-6f0e-da01-865f99a21f07-button" class="content content--spaced" id="867dceb1-e661-6f0e-da01-865f99a21f07-region">
(...)
</section>
 ``` 
**Suggestion**
Ensure ARIA roles, states, and properties are valid. Many ARIA attributes can only be used on certain elements. Some can only be used in combination with other attributes. Additionally, if an attribute references an ID, that ID must be present in the page.

To detect many of these errors automatically, use the Nu Html Checker: https://validator.w3.org/nu/

**Compliant Code Example** 
 ```
<section aria-labelledby="867dceb1-e661-6f0e-da01-865f99a21f07-button" class="content content--spaced" id="867dceb1-e661-6f0e-da01-865f99a21f07-region">
(...)
</section>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by jcfranco** [comment]
This should no longer be an issue with the latest a11y updates to `calcite-block` and `calcite-block-section`.

![Screenshot 2023-11-07 at 10 09 22 AM](https://github.com/Esri/calcite-design-system/assets/197440/fa83bd0d-a2e6-4969-89d4-45342f9902fd)

![Screenshot 2023-11-07 at 10 10 18 AM](https://github.com/Esri/calcite-design-system/assets/197440/c1c2ef6d-ec85-4686-855e-bd80010e62a8)
